scripts/regression_test/regression_test.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           git config --global user.name "Azure CLI Team"
           git remote add azclibot https://azclibot:${GITHUB_TOKEN}@github.com/azclibot/azure-cli.git
 
-          git checkout -b bump_version_$(Build.BuildId)
+          git checkout -b regression_test_$(Build.BuildId)
```

Comment:
This line is not updated:

https://github.com/Azure/azure-cli/blob/e773a7147edb6b7661a4d6cd002a31b4bbfa0710/scripts/regression_test/regression_test.yml#L164

leading to error when creating PR:

```
{
  "message": "Validation Failed",
  "errors": [
    {
      "resource": "PullRequest",
      "field": "head",
      "code": "invalid"
    }
  ],
  "documentation_url": "https://docs.github.com/rest/pulls/pulls#create-a-pull-request"
}
```